client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -185,6 +191,24 @@ export class PluginsListHeader extends PureComponent {
 					</Button>
 				</ButtonGroup>
 			);
+
+			if ( isEnabled( 'manage/plugins/upload' ) ) {
+				const uploadUrl = '/plugins/upload' + ( this.props.selectedSiteSlug ? '/' + this.props.selectedSiteSlug : '' );
+
+				rightSideButtons.push(
+					<ButtonGroup key="plugin-list-header__buttons-upload">
+						<Button
+							compact
+							href={ uploadUrl }
+							onClick={ this.onUploadLinkClick }
+							aria-label={ translate( 'Upload plugin', { context: 'button label' } ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **68** times:
`translate( 'Upload Plugin' )` *ES Score: 13*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/add-plugin-upload-button-to-plugins-list)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.